test/integration/pom.xml

```diff
@@ -210,22 +210,184 @@
 											<value>com.liferay.faces.test.selenium.IntegrationTestSuiteListener</value>
 										</property>
 									</properties>
+									<excludes>
+										<exclude>**/showcase/**</exclude>
+									</excludes>
 									<includes>
 										<include>**/*Test.java,**/Test*.java,**/*TestCase.java,**/*Tester.java</include>
 									</includes>
 									<trimStackTrace>false</trimStackTrace>
-									<!--
-										When specifying alloy tests, no other tests from this execution will be run,
-										so do not fail when no tests are specified.
-									-->
-									<failIfNoSpecifiedTests>false</failIfNoSpecifiedTests>
+								</configuration>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+				<pluginManagement>
+					<plugins>
+						<!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself. -->
+						<plugin>
+							<groupId>org.eclipse.m2e</groupId>
```

Comment:
@stiemannkj1 This is already merged and backported but is the org.eclipse.m2e section necessary? Those sections were removed from all of the project poms when the project moved from the monolithic Git repository to the separate repositories.